options/locale/locale_en-US.ini

```diff
@@ -61,103 +61,103 @@ cancel = Cancel
+requite_db_desc = Gitea requires MySQL, PostgreSQL, MSSQL, SQLite3 or TiDB.
 db_title = Database Settings
 db_type = Database Type
 host = Host
-user = User
+user = Username
 password = Password
 db_name = Database Name
-db_helper = Please use the INNODB engine with utf8_general_ci charset for MySQL.
-ssl_mode = SSL Mode
+db_helper = Note to MySQL users: please use the InnoDB storage engine and the 'utf8_general_ci' character set.
+ssl_mode = SSL
 path = Path
-sqlite_helper = The file path to the SQLite3 or TiDB database. <br>Please use the absolute path when you start as service.
-err_empty_db_path = SQLite3 or TiDB database path cannot be empty.
-err_invalid_tidb_name = TiDB database name does not allow characters "." and "-".
-no_admin_and_disable_registration = You cannot disable registration without creating an admin account.
-err_empty_admin_password = Admin password cannot be empty.
-
-general_title = General Application Settings
-app_name = Application Name
-app_name_helper = You can put your organization name here.
+sqlite_helper = File path for the SQLite3 or TiDB database.<br>Enter an absolute path if you run Gitea as a service.
+err_empty_db_path = The SQLite3 or TiDB database path cannot be empty.
+err_invalid_tidb_name = The TiDB database name cannot contain the '.' and '-' characters.
+no_admin_and_disable_registration = You cannot disable user self-registration without creating an administrator account.
+err_empty_admin_password = The administrator password cannot be empty.
+
+general_title = General Settings
+app_name = Site Title
+app_name_helper = Enter your group name here.
 repo_path = Repository Root Path
-repo_path_helper = All remote Git repositories will be saved to this directory.
-lfs_path = LFS Root Path
-lfs_path_helper = Files stored with Git LFS will be stored in this directory. Leave empty to disable LFS.
-run_user = Run User
-run_user_helper = The user must have access to Repository Root Path and run Gitea.
-domain = Domain
-domain_helper = This affects SSH clone URLs.
+repo_path_helper = Remote Git repositories will be saved to this directory.
+lfs_path = Git LFS Root Path
+lfs_path_helper = Files tracked by Git LFS will be stored in this directory. Leave empty to disable.
+run_user = Run As Username
+run_user_helper = Enter the operating system username that runs Gitea. Note that this user must have access to the repository root path.
+domain = SSH Domain
+domain_helper = Domain or host address for SSH clone URLs.
 ssh_port = SSH Port
-ssh_port_helper = Port number which your SSH server is using. Leave it empty to disable.
+ssh_port_helper = Port number your SSH server listens on. Leave empty to disable.
 http_port = HTTP Port
-http_port_helper = Port number which application will listen on.
-app_url = Application URL
-app_url_helper = This affects HTTP/HTTPS clone URL and some email notifications.
+http_port_helper = Port number that the Gitea web interface will listen on.
```

Comment:
i propose to amend: "… in its execution context (e.g. a container)." to clarify that gitea this is the prt that gitea actually exposes in contrast to a value that is used to assemble strings like a resource url.

Reply:
@funkyfuture I think the new phrasing is pretty self-contained. `…will listen on` doesn't leave much room for interpretation? I could lead with `TCP/IP port number` to make it sound even more physical. I'm pretty sure there's no doubt that Gitea will actually bind and listen on a port on the system you are running it on. Do any others have a strong opinion on this?

Reply:
> I'm pretty sure there's no doubt that Gitea will actually bind and listen on a port on the system you are running it on.

for my sake it's essential to not get the impression that the reachable port is meant for the third time. ;-) my wrong impression obviously stemmed from the fact that other fields surrounding this one are rather 'virtual' (and reachable by the user) than 'physical'. that's worth to explicit in the clearest possible manner without assuming a likely interpretation.

Reply:
and there's no helpful log message when entering a wrong value here…

Reply:
Will push an updated branch soon. I've tried to make these fields even less ambiguous. I can see where you're coming from.